client/src/elements/schema-editor/schema-editor-object.html

```diff
@@ -26,7 +26,7 @@
 
   <div
     if.bind="schema.properties"
-    repeat.for="propertyName of schema.properties | keys"
+    repeat.for="propertyName of schema.properties | keys | filter: {remove: '__observers__'}"
```

Comment:
Oh, was that a property that now got added because of some new code or did you only see the issue thanks to some logging?

Reply:
only detected it now, was also not a problem because it never rendered anything as there is no type to be found from it...
but it's cleaner like this.